Dev7/TriangleMain.java

```diff
@@ -0,0 +1,30 @@
+import java.util.InputMismatchException;
+
+public class TriangleMain {
+  public static void main(String[] args) {
+    InputSides input = new InputSides();
+    double[] array = input.inputSides();
+    Triangle triangle = new Triangle(array);
+    try {
```

Comment:
Ok, you have created right hierarchy for triangles, now let move forward - all these try/catch in one main is not good style, please, create builder class for each triangle class and handle exceptions in these builders. After you have builders ideally you should combine it to the chain of responsibility